build/libswoc.m4

```diff
@@ -51,26 +51,29 @@ AC_ARG_WITH(libswoc, [AS_HELP_STRING([--with-libswoc=DIR],[use a specific libswo
     AC_MSG_RESULT([not found])
   fi
 
-if test "$has_libswoc" != "no"; then
-  saved_ldflags=$LDFLAGS
-  saved_cppflags=$CPPFLAGS
+  if test "$has_libswoc" != "no"; then
+    saved_ldflags=$LDFLAGS
+    saved_cppflags=$CPPFLAGS
 
-  SWOC_LIBS=-ltsswoc
-  if test "$libswoc_base_dir" != "/usr"; then
-    SWOC_INCLUDES=-I${swoc_include}
-    SWOC_LDFLAGS=-L${swoc_ldflags}
+    SWOC_LIBS=-lswoc
```

Comment:
Can this value be set via the pkg-config file instead??

Reply:
Updated.